SDKPackageManager.ps1

```diff
@@ -0,0 +1,31 @@
+param([Parameter(Mandatory=$true)][string]$arch, [Parameter(Mandatory=$true)][string]$config, [Parameter(Mandatory=$true)][string]$task)
+
+$currentLocation = Get-Location
+$packagePath = "$currentLocation\BuildOutput\$config\$arch\"
+
+if(!$task.CompareTo("i"))
+{
+	write-host "installing packages"
+	add-appxpackage "$packagePath\Microsoft.WindowsAppRuntime.Framework\Microsoft.WindowsAppRuntime.Framework.msix"
+	add-appxpackage "$packagePath\DynamicDependency.DataStore.Msix\DynamicDependency.DataStore.msix"
+	add-appxpackage "$packagePath\DynamicDependencyLifetimeManager.Msix\DynamicDependencyLifetimeManager.msix"
+	add-appxpackage "$packagePath\PushNotificationsLongRunningTask.Msix\PushNotificationsLongRunningTask.msix"
+}
+elseif(!$task.CompareTo("u"))
+{
+	taskkill /IM PushNotificationsLongRunningTask.exe /F
+	taskkill /IM DynamicDependency.DataStore.exe /F
```

Comment:
Why?

This only runs if DynDep APIs are called in an AppContainer. Otherwise DynDep uses `ApplicationDataManager.CreateForPackageFamily()` which is an inproc API (no helper process needed).